Apps/FCM/src/screens/InternalSettings.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import { View, Text, ScrollView } from 'react-native';
+import { TextField } from '../components/TextField';
+import { FilledButton } from '../components/Button';
+import {styles as settingsStyles} from '../styles/stylesheet';
+
+const InternalSettings: React.FC = () => {
+    const [apiHost, setApiHost] = useState('');
+    const [cdnHost, setCdnHost] = useState('');
+    const [flushAt, setFlushAt] = useState('');
+    const [flushInterval, setFlushInterval] = useState('');
+
+    const handleSave = () => {
+
+    };
+    
+
+    return (
+        <View style={settingsStyles.container}>
+            <ScrollView contentContainerStyle={settingsStyles.content}>
+                <View style={settingsStyles.section}>
+                    <TextField
+                        style={settingsStyles.textInputContainer}
+                        label="CDN Host"
+                        value={cdnHost}
+                        contentDesc="CDN Host Input"
+                        placeholder='CDN Host'
+                        onChangeText={text => setCdnHost(text)}
+                        // textInputRef={siteIdRef}
+                        textInputProps={{
+                            autoCapitalize: 'none',
+                            keyboardType: 'default',
+                        }}
+                    />
+                    <TextField
+                        style={settingsStyles.textInputContainer}
+                        label="API Host"
+                        value={apiHost}
+                        placeholder='API Host'
+                        contentDesc="API Host Input"
+                        onChangeText={text => setApiHost(text)}
+                        // textInputRef={siteIdRef}
+                        textInputProps={{
+                        autoCapitalize: 'none',
+                        keyboardType: 'default',
+                        }}
+                    />
+                    <TextField
+                        style={settingsStyles.textInputContainer}
+                        label="Flush Interval"
+                        value={flushInterval}
+                        contentDesc="Flush Interval Input"
+                        onChangeText={text => setFlushInterval(text)}
+                        // textInputRef={siteIdRef}
+                        // getNextTextInput={() => ({ ref: apiKeyRef, value: cdpApiKey })}
+                        textInputProps={{
+                        autoCapitalize: 'none',
+                        placeholder: '30',
+                        keyboardType: 'default',
```

Comment:
Is this deliberate for "Flush Interval" and "Flush At"?

Reply:
I’m not sure what you’re referring to—are you talking about the placeholder?

If so, you’re correct. I added a number as a placeholder that matches the default configuration. It’s not necessary, but I included it to align with the UI that Scott shared in the [Loom video](https://www.loom.com/share/f5ca9eedf32e4b708e4cadff69545e12?sid=6b0cc64c-3135-4202-8fcf-82c576dc51da).

Reply:
Makes sense Aman. I'm referring to `keyboardType: 'default'` here, as the keyboard opened for these fields will be non-numeric.

Reply:
Ah ! Right ! I'll make the change.